src/react/components/_academics/EnrollmentLinks/EnrollmentLinks.js

```diff
@@ -30,18 +30,19 @@ const EnrollmentInstructionLinks = (props) => {
   const currentTermId = props.instruction.termId;
   const acknowledgement = forTermId(props.calgrantAcknowledgements, currentTermId);
```

Comment:
Sorry to comment on something not related to the PR, but I didn't get the chance to review this in the first place.

Why are we calling this `forTermId` instead of `findByTermId`?

Reply:
No reason, I'm open to changing it in the future